src/start-express-app.ts

```diff
@@ -54,16 +55,13 @@ export default async function(options: N9NodeRouting.Options): Promise<N9NodeRou
 		const formatLogInJSON: boolean = global.n9NodeRoutingData.formatLogInJSON;
 
 		if (formatLogInJSON) {
-			const totalResponseTime = tokens['total-response-time'](req, res);
 			return JSON.stringify({
```

Comment:
We should use `fast-safe-stringify` here as well

Reply:
I don't see why, here we are master of each field. I deliberately kept this `JSON.stringify`  for performances matter.